appinventor/components/src/com/google/appinventor/components/runtime/Ev3UltrasonicSensor.java

```diff
@@ -113,7 +109,8 @@ public double GetDistance() {
   }
 
   private double getDistance(String functionName) {
-    double distance = readInputSI(functionName, 0, sensorPortNumber, SENSOR_TYPE, mode);
+    double distance = readInputSI(
```

Comment:
I would prefer wrapping after the last comma that occurs < 100 characters, rather than putting all of the arguments on the new line (personal preference here, the guide allows what you're doing).